network-api/networkapi/buyersguide/templates/fragments/product_criterion_primary_info.html

```diff
@@ -4,11 +4,11 @@
   <h3 class="pni-product-smaller-body-bold">{{ label }}</h3>
 
   <p class="rating pni-product-smaller-body mb-0">
-    {% if value == "U" %}
+    {% if value == "CD" %}
       {% if show_value_as_symbol %}
       <img src="{% static "_images/buyers-guide/score/icon-unknown.svg" %}" alt="{% trans "Can’t determine" context "Unknown rating" %}">
       {% else %}
-      {% trans "Unknown" context "Unknown rating" %}
+      {% trans "Can't Determine" context "Unknown rating" %}
```

Comment:
```suggestion
      {% trans "Can’t Determine" context "Unknown rating" %}
```

Also if the capital D is not required, we could automatically reuse the translation from two lines above since it’s the exact same context.

Reply:
Oops! Very good point. Thanks for pointing this out!